fastlane/spec/fast_file_spec.rb

```diff
@@ -138,10 +146,12 @@
         ff = Fastlane::FastFile.new('./spec/fixtures/fastfiles/Fastfile1')
         ff.runner.execute(:deploy)
         expect(File.exist?('/tmp/fastlane/before_all')).to eq(true)
+        expect(File.exist?('/tmp/fastlane/before_each_deploy')).to eq(true)
         expect(File.exist?('/tmp/fastlane/deploy')).to eq(true)
         expect(File.exist?('/tmp/fastlane/test')).to eq(false)
         expect(File.exist?('/tmp/fastlane/after_all')).to eq(true)
         expect(File.read("/tmp/fastlane/after_all")).to eq("deploy")
+        expect(File.exist?('/tmp/fastlane/after_each_deploy')).to eq(true)
```

Comment:
Ah, I see what you mean. We'll need a fastfile with many lanes and make sure that each have a file